application/common/mail/reset/on-behalf.php

```diff
@@ -0,0 +1,15 @@
+Hi there,
+
+<?php echo \yii\helpers\Html::encode($name); ?> recently requested a password change for their
+<?php echo \Yii::$app->params['appName']; ?> account and they have requested your assistance.
```

Comment:
So would `appName` refer to the overall system being affected by these password resets, or does it refer to this password reset system itself?  I assumed `appName` would be something like "[System] Password Manager", not just "[System]", but the way it's used here seems to indicate otherwise.